### DISCUSSION TITLE
Is it fine to use await.import instead of await deep.import?

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#28] [Q&A]
This code is working when I use it in handler:
```ts
async () => {
  const { createSerialOpeation } = await import('@deep-foundation/deeplinks/imports/gql/index.js');
  console.log(12312321)
}
```
And I do not want to use `await deep.import` because I will loose code autocompletion.

Let me show you what I can do when I install my package and open it in vscode and use await deep.import:
![image](https://github.com/deep-foundation/Discussions/assets/66206278/558df76e-46f8-4cce-b038-287e8610c2d2)

**Comment by konard** [comment]
We need to discuss it with @ivansglazunov 

When we will use bundler, it will be impossible to use regular `await import`. Or we may pass an additional `import` argument into this function in handlers that will be equal to `await deep.import`. But it may still break autocompletion.

**Reply by FreePhoenix888** [reply]
I would like to have autocompletion
